src/main/java/eladkay/hudpixel/util/autotip/command/AutotipCommand.kt

```diff
@@ -79,7 +78,7 @@ class AutotipCommand : HpCommandBase() {
                             "year", "yearly" -> Stats.printBetween("01-01-" + Year.now().value,
                                     "31-12-" + Year.now().value)
                             "all", "total", "life", "lifetime" -> Stats.printBetween("25-06-2016", FileUtil.date)
```

Comment:
What is the "25-06-2016" for? Why not just log when they started or at the beginning?